test cases/shader/1 basic/test.json

```diff
@@ -0,0 +1,5 @@
+{
+  "installed": [
+    {"type": "file", "file": "usr/bin/basic.spv"}
```

Comment:
Is this really an executable program runnable from the command line?

Reply:
Not directly, but they can loaded by a program at runtime. It is theoretically an executable just not on the cpu, but the gpu.